src/lib/merkle_ledger/base_ledger_intf.ml

```diff
@@ -76,6 +76,10 @@ module type S = sig
   (** Get all of the tokens for which a public key has accounts. *)
   val tokens : t -> key -> token_id_set
 
+  val next_available_token : t -> token_id
```

Comment:
might be worth a comment about what it means for a tid to be available, and about the ordering on tids